### PR TITLE
added self-serve button to and reformatted the Earthly CI sign up page.

### PR DESCRIPTION
**Pull request by thtmnisamnstr**

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
Preview URL: https://642374df6dac1314e08115c2--ephemeral-narwhal-3ac3b2.netlify.app/signup/earthly-ci/

File: website/signup/earthly-ci/index.html [hunk 1]
```diff
@@ -14,96 +14,115 @@
...
+            </div>
+          </div>
+          <div class="py-2 lg:py-6">
+            <div class="text-xl lg:text-2xl text-slate-500 pb-1">
+              <span class="font-bold">If you have less than 5 users</span>, click here to go to our guide to self-serve Earthly CI.
+            </div>
+            <div class="text-base lg:text-lg pb-3">
+              This will take you to our documentation and give you steps to sign up for and onboard onto Earthly CI.
+            </div>
+            <form id="earthly-ci-self-serve-form" method="GET" action="https://docs.earthly.dev/earthly-cloud/earthly-ci#getting-started">
```
Comment (vladaionescu) [inline website/signup/earthly-ci/index.html:56]: I would link to the beginning of the page here, to also capture the introduction.

```suggestion
            <form id="earthly-ci-self-serve-form" method="GET" action="https://docs.earthly.dev/earthly-cloud/earthly-ci">
```

**Comment by thtmnisamnstr** [comment]
@mirho Please review this for styling and layout tomorrow. If you have changes, feel free to make them directly to the [`gavin/ci-self-serve`](https://github.com/earthly/website/tree/gavin/ci-self-serve) branch.

@adamgordonbell Please review this tomorrow and verify that I've added the correct tracking tag (and haven't broken the previous one you added).

@vladaionescu Please review this tomorrow for content and styling.

File: website/signup/earthly-ci/index.html [hunk 2]
```diff
@@ -14,96 +14,115 @@
...
+              <button
+                  class="pt-3 pb-3 text-xl text-center text-white font-medium bg-green-500 hover:bg-green-700 rounded-lg w-full lg:w-80 inline-block"
+                  type="submit" >
+                  Self-Serve
```
Comment (vladaionescu) [inline website/signup/earthly-ci/index.html:60]:
```suggestion
                  Get Started Now
```

File: website/signup/earthly-ci/index.html [hunk 3]
```diff
@@ -14,96 +14,115 @@
...
+              </div>
+              <div class="flex pb-2">
+                  <img src="../../assets/svg/done-check.svg" alt="Checked bullet" class="" />
+                  <div class="px-2">Remote build runners built-in</div>
+              </div>
+            </div>
+          </div>
+          <div class="py-2 lg:py-6">
+            <div class="text-xl lg:text-2xl text-slate-500 pb-1">
+              <span class="font-bold">If you have less than 5 users</span>, click here to go to our guide to self-serve Earthly CI.
```
Comment (vladaionescu) [inline website/signup/earthly-ci/index.html:51]:
```suggestion
              <span class="font-bold">If you have less than 5 users</span>, follow the self-guided onboarding.
```

File: website/signup/earthly-ci/index.html [hunk 4]
```diff
@@ -14,96 +14,115 @@
...
+                  type="submit" >
+                  Self-Serve
+              </button>
+            </form>
+          </div>
+          <div class="formkit-container py-2 lg:py-6">
+              <div class="formkit-main">
+                  <div>
+                      <div class="text-xl lg:text-2xl text-slate-500 pb-1">
+                        <span class="font-bold">If you have 5+ users</span>, sign up here, and we'll contact you shortly to begin onboarding.
```
Comment (vladaionescu) [inline website/signup/earthly-ci/index.html:68]:
```suggestion
                        <span class="font-bold">If you have 5+ users</span>, sign up here, and we'll contact you shortly to begin assisted onboarding.
```

**Review by vladaionescu — COMMENTED** [review]
This makes sense - the distinction between the two paths seems pretty good.

If it's not too difficult for @mirho, I think that a slight improvement might be to have everything above the fold (e.g. having the two paths laid out horizontally rather than vertically). Not critical though - this works like this too.

I left some suggestions on improving the text a bit.

**Comment by mirho** [comment]
@thtmnisamnstr @vladaionescu will take a look at this. From what I can see it needs a bit design work.

Do we want to prioritize self-serve? Or are they equal in terms of importance? Its good to have a goal there, then we can design with that in mind.

**Comment by thtmnisamnstr** [comment]
@mirho Prioritize self-serve.

**Review by adamgordonbell — APPROVED** [review]
LGTM!

**Comment by mirho** [comment]
@thtmnisamnstr cool will work on the design

**Comment by mirho** [comment]
I spent some time tweaking the design, and also the wording a bit. See screenshot below

image

**Comment by vladaionescu** [comment]
Looks great!

**Pull request merged**